test/java/seedu/addressbook/TestDataHelper.java

```diff
@@ -59,19 +61,14 @@ public Fees fees (int seed) throws Exception {
         return new Fees(value, dueDate);
     }
 
-    /** Test exam for testing**/
+    /** Test statistics for testing**/
     public AssignmentStatistics stat() throws Exception {
-        String subjectName = "Spanish";
```

Comment:
Methods should sound like verbs

Reply:
Help to add "make" for the other object creators as well. eg. makeStat, makeAdam